src/app/components/hris/employees/employee-profile/accordions/accordion-documents/accordion-additional-documents/accordion-additional-documents.component.ts

```diff
@@ -120,6 +121,21 @@ export class AccordionDocumentsCustomDocumentsComponent {
       next: data => {
         this.sharedAccordionFunctionality.customFieldsDocuments = data.filter((data: CustomField) => data.category === this.sharedAccordionFunctionality.category[3].id);
         this.checkCustomDocumentsInformation();
+        this.checkArchived(data);
+      }
+    })
+  }
+
+  checkArchived(fields: any) {
+    var index = 0;
+    fields.forEach((field: { category: number; status: any; }) => {
+      index++;
+      if (field.status == -1 && field.category == 3) {
```

Comment:
Refactor: the if statement has no errors, however, it will also yield no compile errors with field.status = -1, 

if we were to change it to 
-1 == field.status 
then the compiler will throw an error when someone accidently removes one equals. Doing it this way will help mitigate that risk.